src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -459,7 +459,8 @@ private string UFormatDateString(DateTime dateTime)
                             break;
 
                         case 'V':
-                            sb.Append((dateTime.DayOfYear / 7) + 1);
+                            GregorianCalendar calendar = new GregorianCalendar();
```

Comment:
I believe we should follow CoreFX and should be `CurrentCulture` based not ISO based.
/cc @SteveL-MSFT